src/Razor/src/Microsoft.VisualStudio.RazorExtension/language-configuration.json

```diff
@@ -30,14 +30,14 @@
   },
   "onEnterRules": [
     {
-      "beforeText": "<(?!(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr))([_:\\w][_:\\w-.\\d]*)([^/>]*(?!\\/)>)[^<]*$",
+      "beforeText": "<(?!(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr))([_:\\w][_:\\w-.\\d]*)([^/>]*(?!.*(?<!=>)\\/)>)[^<]*$",
       "afterText": "^<\\/([_:\\w][_:\\w-.\\d]*)\\s*>",
       "action": {
         "indent": "indentOutdent"
       }
     },
     {
-      "beforeText": "<(?!(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr))(\\w[\\w\\d]*)([^/>]*(?!\\/)>)[^<]*$",
+      "beforeText": "<(?!(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr))(\\w[\\w\\d]*)([^/>]*(?!.*(?<!=>)\\/)>)[^<]*$",
```

Comment:
even though this fixes the test, it might not be very performant. will try to improve it a little better.

Reply:
I swear I tried a negative look behind and couldn't get it to work. Nice job!

<sub>you now officially own all of our regexes :P</sub>

Reply:
You are now the regex grandmaster. 

![GrandMasterEntrranceGIF](https://github.com/dotnet/razor/assets/475144/12483d28-b3b4-4c78-9cf3-914eb186bf52)